numpy/__init__.pyi

```diff
@@ -522,15 +522,25 @@ def array(
     copy: bool = ...,
     subok: bool = ...,
     ndmin: int = ...,
+    like: ArrayLike = ...,
 ) -> ndarray: ...
 def zeros(
-    shape: _ShapeLike, dtype: DtypeLike = ..., order: Optional[str] = ...
+    shape: _ShapeLike,
+    dtype: DtypeLike = ...,
+    order: Optional[str] = ...
+    like: ArrayLike = ...,
```

Comment:
```suggestion
    *,
    like: ArrayLike = ...,
```

Reply:
Thanks @BvB93 , I couldn't find whether the keyword-only marker is supported in `.pyi` files, so I assumed it wasn't, but now I see `reshape` and `resize` had it, my mistake, but it's fixed now.